contracts/utils/contracts/src/v08/LibSafeMathV08.sol

```diff
@@ -0,0 +1,169 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+
+  Copyright 2020 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.8;
+
+import "./errors/LibRichErrorsV08.sol";
+import "./errors/LibSafeMathRichErrorsV08.sol";
+
+library LibSafeMathV08 {
```

Comment:
`LibSafeMathV08` is likely not needed in 0.8 as overflow/underflow checks are done by default.

Looks like there are some non-safe math related utility functions like `max256` though and it's also used by `LibMath` which complicates things. 🤔